Auth0/SafariWebAuth.swift

```diff
@@ -153,6 +158,7 @@ class SafariWebAuth: WebAuth {
             entries["nonce"] = self.nonce
         }
         self.parameters.forEach { entries[$0] = $1 }
+        self.parameters = [:]
```

Comment:
@hzalaz Can I suggest there should be a method to reset parameters, if you are using a common instance of `WebAuth`, parameters are cumulative with no way to remove them. 
This is most relevant with the requirement of connection specific params like `connectionScopes` and `login_hint`.

You can of course create a new instance each time but it is less expensive to `[:]` than it is init a new object. 

Thoughts?